service/src/main/kotlin/fi/espoo/evaka/messaging/MessageController.kt

```diff
@@ -326,7 +327,18 @@ class MessageController(
             }
     }
 
-    data class PostMessagePreflightBody(val recipients: Set<MessageRecipient>)
+    data class PostMessageFilters(
+        val yearsOfBirth: List<Int> = listOf(),
+        val serviceNeedIds: List<Id<*>> = listOf(),
```

Comment:
Variable name says they are service need ids, but type says it accepts any id (person, placement, daycare, etc...)?